gravitee-apim-console-webui/src/entities/portal/portalSettings.ts

```diff
@@ -229,5 +229,15 @@ export interface PortalSettingsPortalNext {
     title?: string;
     subtitle?: string;
     enabled?: boolean;
+    primaryButton?: BannerButton;
+    secondaryButton?: BannerButton;
   };
 }
+
+export interface BannerButton {
+  enabled?: boolean;
+  label?: string;
+  target?: string;
+  type?: 'EXTERNAL';
+  visibility?: 'PUBLIC' | 'PRIVATE';
```

Comment:
Why all of the fields are optional?
Can we have at least one of them mandatory?